lms/djangoapps/teams/tests/test_views.py

```diff
@@ -426,23 +443,44 @@ def test_full(self):
             topic_id='great-topic',
             country='CA',
             language='fr'
-        ))
+        ), user=creator)
 
         # Remove date_created and discussion_topic_id because they change between test runs
         del team['date_created']
         del team['discussion_topic_id']
-        self.assertEquals(team, {
+
+        # Since membership is its own list, we want to examine this separately.
+        team_membership = team['membership']
+        del team['membership']
+
+        # Verify that the creating user gets added to the team.
+        self.assertGreater(len(team_membership), 0)
```

Comment:
Given that this team was just created, there should only be 1 member in the team. Why just not test len == 1?